### DISCUSSION TITLE
Project Rename - What Should It Be?

### DISCUSSION THREAD

**Discussion by simplexidev** [tom-corwin/LibUISharp#41] [Q&A] [answered]
So this project started, ended, and started again as just bindings for [libui](https://github.com/andlabs/libui). Now, the plan is to add an audio API, graphical drawing support, and more, allowing for robust applications, as well as games.

If there are any suggestions for a name, please add a comment and it very well might be it!

**Comment by Shadowblitz16** [comment]
- SharpForms ( This probably implies its a cross platform implementation of winforms )
- CorWinForms ( same as above XD)
- TomForms
- UISharp
- SharpUI
- Corwin.UI
- Corwin.Forms

**Reply by simplexidev** [reply]
Almost makes me want to rename my account now, rather than later. Plan is to rename this account to `SimplexiDev`, so I might just go and do that sooner than later, and make the primary namespace, well, `SimplexiDev`.

> * CorWinForms ( same as above XD)

That was a good one 😆 I needed that laugh. Never thought it would work as well as it did together.

BTW, if I don't rename my account first, I'm probably going to lean more towards `Corwin.UI`. I'd like to avoid the `Forms` namespaces, as it will be misleading as to associate with WinForms. 

And [sharpui](https://github.com/benpye/sharpui) is already in use, for you guessed it, libui bindings XD

**Comment by simplexidev** [comment, marked as answer]
Well, GitHub Support changed my username from `@tom-corwin` to @simplexidev for me, so I guess the name change is going into #39. If there are any questions, feel free to comment on this discussion.
